src/test/java/org/jenkinsci/plugins/badge/actions/EmbeddableBadgeConfigsActionTest.java

```diff
@@ -0,0 +1,53 @@
+package org.jenkinsci.plugins.badge.actions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+
+import hudson.model.Job;
+import hudson.model.Run;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class EmbeddableBadgeConfigsActionTest {
+
+    EmbeddableBadgeConfigsAction embeddableBadgeConfigsAction;
+
+    @BeforeEach
+    void setUp() {
+        Run mockRun = Mockito.mock(Run.class);
+        Job mockJob = Mockito.mock(Job.class);
+        Mockito.when(mockRun.getParent()).thenReturn(mockJob);
```

Comment:
Not needed and not used.  Please delete these lines since the tests run as expected without them
```suggestion
```